src/js/modules/articles/components/ArticleHeader.js

```diff
@@ -58,13 +61,17 @@ const styles = {
 
 // TODO: make selector for dateline
 
-const ArticleHeader = ({ classes, contributors, dateline, headline, section }) => {
+const ArticleHeader = ({
+                         classes,
+                         article: { contributors, dateline, title },
```

Comment:
What does this mean? Is article an object with keys `contributors` `dateline` and `title`? Or is it just like `article.contributors` but with shorthand?